.circleci/config.yml

```diff
@@ -170,12 +170,12 @@ jobs:
           command: |
             echo "mode: set" > coverage.out
             grep -h -v "mode: set" cov_*.part >> coverage.out
-            go tool cover -html=coverage.out -o /tmp/test-results/coverage.html
+            go tool cover -html=coverage.out -o $(TEST_RESULTS_DIR)/coverage.html
```

Comment:
This is similar but not quite the same -- figured it was in scope of this PR because it's another place where we manually define `/tmp/test-results` Didn't actually test this one so hoping CI comes back green and I didn't need quotes 🤞

Reply:
I believe this will fail because $() is a Make-ism and this is bash config. The normal `${TEST_RESULTS_DIR}` or `$TEST_RESULTS_DIR should work fine here.

Reply:
Ah, yes, thank you! There's examples of this in other command blocks in this file so doh 🤦‍♀